From 097f36d4b5d7256c3623dcbaa8e44bfffc397675 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Sun, 17 Mar 2019 18:15:12 +0200
Subject: [PATCH 062/386] dts: mcbin: add in-band-status to MACCHIATObin single
 shot 10G ports

10G ports doesn't have PHY's so they should be managed in-band.

Change-Id: I3b3c988ed24f2d11d265391c7d959ecdd68f1eca
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/5855
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Yoav Gvili <ygvili@marvell.com>
Reviewed-by: Marcin Wojtas <Marcin.Wojtas@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin-single-shot.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-single-shot.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-single-shot.dts
index 5db80122795d..8da391cea04e 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-single-shot.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-single-shot.dts
@@ -21,6 +21,8 @@
 			ethernet@0 {
 				eth0 {
 					/delete-property/phy;
+					managed = "in-band-status";
+					sfp = <&sfp_eth0>;
 				};
 			};
 			/delete-node/cp0_xmdio;
@@ -32,6 +34,8 @@
 			ethernet@0 {
 				eth0 {
 					/delete-property/phy;
+					managed = "in-band-status";
+					sfp = <&sfp_eth1>;
 				};
 			};
 			/delete-node/cp1_xmdio;
-- 
2.17.1

